app/components/subject-viewer.cjsx

```diff
@@ -71,6 +71,10 @@ module.exports = React.createClass
       clearTimeout @signInAttentionTimeout
       @setState loading: true
 
+  componentDidUpdate: (prevProps) ->
```

Comment:
Why place line 75-76 in `componentDidUpdate` instead of `componentWillReceiveProps`?